### PR TITLE
Updates afero lab to handle both legacy and web flow authentication 

### PR DESCRIPTION
**Pull request by marnberg**
Enable configuration with both authenticatorCert and hubby service.

### CODE REVIEW

File: AferoSwiftSDK/Core/Models/Account.swift [hunk 1]
```diff
@@ -259,9 +259,9 @@ public struct UserAccount {
         
         public init(userId: String, credentialId: String?, credential: Credential? = nil, firstName: String? = nil, lastName: String? = nil, name: String? = nil, accountAccess: [AccountAccess]? = nil, sharingAccountAccess: SharingAccountAccess? = nil, partnerAccess: [PartnerAccessType]? = nil, tos: [ToS]? = nil, lastUsed: Date? = nil) {
 
-            if firstName == nil && lastName == nil && name == nil {
-                fatalError("Need at least one name field to make a valid user object")
-            }
+//            if firstName == nil && lastName == nil && name == nil {
```
Comment (jrmiddle) [inline AferoSwiftSDK/Core/Models/Account.swift:262]: We can just drop this altogether.

File: Examples/AferoLab/AferoLab.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -878,11 +884,11 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_ENABLE_MODULES = YES;
 				CODE_SIGN_ENTITLEMENTS = AferoLab/AferoLab.entitlements;
-				DEVELOPMENT_TEAM = 534JTUC87X;
+				DEVELOPMENT_TEAM = YVKN778Q2B;
 				INFOPLIST_FILE = AferoLab/Info.plist;
 				IPHONEOS_DEPLOYMENT_TARGET = 9.3;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
-				PRODUCT_BUNDLE_IDENTIFIER = io.afero.AferoLab;
+				PRODUCT_BUNDLE_IDENTIFIER = io.afero.demo.examle;
```
Comment (jrmiddle) [inline Examples/AferoLab/AferoLab.xcodeproj/project.pbxproj:891]: Typo, should be: `...demo.example`

File: Examples/AferoLab/AferoLab/Base.lproj/LaunchScreen.storyboard [hunk 3]
```diff
@@ -22,7 +20,7 @@
                         <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
                         <subviews>
                             <imageView userInteractionEnabled="NO" contentMode="center" horizontalHuggingPriority="251" verticalHuggingPriority="251" image="Startup" translatesAutoresizingMaskIntoConstraints="NO" id="VLC-C0-pOK">
-                                <rect key="frame" x="87" y="210" width="200" height="246"/>
+                                <rect key="frame" x="87.5" y="210.5" width="200" height="246"/>
```
Comment (jrmiddle) [inline Examples/AferoLab/AferoLab/Base.lproj/LaunchScreen.storyboard:23]: No big whup, but did something get nudged? Looks like something was knocked out of alignment.

**Review by jrmiddle — CHANGES_REQUESTED** [review]
One last question—do all of the unit tests pass? They _should_, as I don't see anything here that would bother them, but just checking. Otherwise, with the exception of the typo in the new bundle ID, this looks good!

**Comment by marnberg** [comment]
Thank you @jrmiddle for the feedback. I updated the softhub version to 1.8.0 and fixed the unintended changes

**Pull request merged**